### DISCUSSION TITLE
DBeaver vs SQLiteStudio

### DISCUSSION THREAD

**Discussion by lucausa75** [pawelsalawa/sqlitestudio#5071] [Q&A]
Dear All,

I have recently used an SQLite database and in order to manage it I used DBeaver.
Das ago I have found something more interesting: SQLiteStudio which is working good but I would like to understand some few things.
In DBeaver I use to create some scripts for each database with procedure inside which, If need, can be run: I did not find anything in SQLStudio: is there something similar?
I found only the way to generate snippets but I'm not able to run the query they have inside; please could you help me?

Thanks

**Comment by pawelsalawa** [comment]
In SQLite (not in SQLiteStudio, but in SQLite database itself) there are functions, which you can invoke by executing `SELECT function_name()`.

SQLiteStudio has editor for your own functions - see https://github.com/pawelsalawa/sqlitestudio/wiki/User_Manual#custom-sql-functions
